### DISCUSSION TITLE
Frigate+ Models

### DISCUSSION THREAD

**Discussion by johnmarksilly** [blakeblackshear/frigate#8100] [Ask A Question] [answered]
How accurate are people's models now that Frigate+ can generate them? 

I've annotated around a 100 or so images and built a model, but the accuracy went down significantly for people. Most of the false positives were of random debris in my garage that kept being detected as people, but now it seems everything outside of my house is now a person. 

I'm worried that there aren't enough examples of people to go along with all of the manually annotated images that were originally false. 

Anyone have any suggestions? Should I submit images to Frigate+ that were correct too?

**Comment by NickM-27** [comment, marked as answer]
Depending how many cameras you have, 100 may be way too few. The docs recommend 100 per camera to start. 

Multiple users have reported that the first model had higher false positives and the second model got much better. 

https://github.com/blakeblackshear/frigate/discussions/7932#discussioncomment-7189445

https://github.com/blakeblackshear/frigate/discussions/7932#discussioncomment-7166293

You should be submitting a balance of true and false positives.

**Reply by johnmarksilly** [reply]
Hey Nick I'm going to close this discussion since I ultimately found the sub discussion talking about these specifically. Thanks for reaching out! I've been training while I've been on vacation... lol

Also, does submitting positive training through the UI send a masked image to be used in the model? Does it include all existing labels. How does that aspect work? I should maybe read the new beta docs.

**Reply by NickM-27** [reply]
> Also, does submitting positive training through the UI send a masked image to be used in the model? Does it include all existing labels. How does that aspect work? I should maybe read the new beta docs.

Not sure what you mean by masked image. No matter what you submit you need to go into the frigate+ webUI and finish annotating all objects and verify the image to save

**Reply by johnmarksilly** [reply]
Ah okay. I thought submitting the positive "this is a person" image would crop the image of the person and send it to frigate+ already verified. Would this also work for training since Frigate crops the motion boxes to send to the detector anyway, or it has to train on the full frame?

**Reply by NickM-27** [reply]
Frigate+ takes the full image and divides it into regions based on the annotations and its internal logic, similarly Frigate itself only sends parts of the image based on where motion is detected.
